docs/genai/api/python.md

```diff
@@ -28,6 +28,57 @@ pip install onnxruntime-genai
 import onnxruntime_genai
 ```
 
+## Config class
+
+If the config needs to be modified at runtime to change providers/set provider options, this object can be first created and used, then a model can be created from this config object.
```

Comment:
It is perhaps obvious but maybe worth stating that if the user does not create the object, then the config is used directly from the file in the model folder